deploy/olm-catalog/manifests/assisted-service-operator.clusterserviceversion.yaml

```diff
@@ -86,6 +86,20 @@ metadata:
                 "rootFSUrl": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.10/4.10.3/rhcos-4.10.3-aarch64-live-rootfs.aarch64.img",
                 "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.10/4.10.3/rhcos-4.10.3-aarch64-live.aarch64.iso",
                 "version": "410.84.202201251210-0"
+              },
+              {
+                "cpuArchitecture": "x86_64",
+                "openshiftVersion": "4.11",
+                "rootFSUrl": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/latest-4.11/rhcos-live-rootfs.x86_64.img",
+                "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/latest-4.11/rhcos-live.x86_64.iso",
```

Comment:
Why not using a specific version e.g. [rhcos-4.11.0-0.nightly-2022-04-16-163450-x86_64-live.x86_64.iso](https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/latest-4.11/rhcos-4.11.0-0.nightly-2022-04-16-163450-x86_64-live.x86_64.iso)?

Reply:
The specific nightly URL (rhcos-4.11.0-0.nightly-2022-04-16-163450-x86_64-live.x86_64.iso) under latest-4.11 changes (every nightly).
`https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/latest-4.11/rhcos-live.x86_64.iso` should stay the same.

Reply:
That's right
in https://github.com/openshift/assisted-service/commit/94ce200b59e8bb3fa0032819a727aa85b532dc22 we used to have https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest-4.10/rhcos-4.10.0-0.nightly-2021-11-29-191648-x86_64-live.x86_64.iso that is long gone

Reply:
I don't like this non-pinned version, but at least it will stay there for a while

Reply:
fwiw, you can use the exact version from the specific folder: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-0.nightly-2021-11-29-191648/
iinm it shouldn't be deleted for a while, but probably still safer to use latest for now

Reply:
I also think the link you suggested is better.
The non-versioned image won't get upgraded as expected and we won't be able to tell exactly which image we're using.

In the image service we also potentially could end up with different replicas using different base images.

Reply:
Done
Please take another look